packages/components/src/components/button-pure/button-pure.tsx

```diff
@@ -153,7 +153,7 @@ export class ButtonPure {
               {...iconProps}
               name={this.icon}
               source={this.iconSource}
-              color={this.isDisabledOrLoading ? 'contrast-medium' : 'primary'}
+              color={this.isDisabledOrLoading ? 'disabled' : 'primary'}
```

Comment:
Since this is essentially an alias for `contrast-medium` it would make sense to align it across the entire codebase in the components' render functions.